source/includes/_invoice.md

```diff
@@ -3973,44 +2449,19 @@ invoiceApi.create_invoice_tags(invoice_id,
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 201 Created
-< Location: http://localhost:8080/1.0/kb/invoices/2cd2f4b5-a1c0-42a7-924f-64c7b791332d/tags
-< Content-Type: application/json
-< Content-Length: 0
-```
-```java
-//First element of the list
-class Tag {
-    org.killbill.billing.client.model.gen.Tag@bd138472
-    tagId: 1bb4b638-3886-4f73-90a5-89eb6d1bcf7f
-    objectType: INVOICE
-    objectId: 45d6f4c5-21be-49b1-99c5-7b0c3c985bf0
-    tagDefinitionId: 00000000-0000-0000-0000-000000000004
-    tagDefinitionName: WRITTEN_OFF
-    auditLogs: []
-}
-```
-```ruby
+```json
 [
    {
       "tagId":"a46cfeb6-e175-42db-be62-7f117326ab4e",
       "objectType":"INVOICE",
       "objectId":"28af3cb9-275b-4ac4-a55d-a0536e479069",
       "tagDefinitionId":"00000000-0000-0000-0000-000000000004",
       "tagDefinitionName":"WRITTEN_OFF",
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?